modules/node-shared/src/main/resources/application.conf

```diff
@@ -184,8 +184,21 @@ metagraphs-sync {
 }
 
 price-oracle {
-  # null means all metagraph ids are allowed
-  allowed-metagraph-ids = null
-  # allowed-metagraph-ids = ["DAG2deV7SKjC2xhTzmpX2ASmvEPzuzkSjXPRcSDy"]
-  min-epochs-between-updates = 0
+  mainnet: {
+    allowed-metagraph-ids = []
+    min-epochs-between-updates = 50
+  }
+  testnet: {
+    allowed-metagraph-ids = null
+    min-epochs-between-updates = 0
+  }
+  integrationnet: {
+    allowed-metagraph-ids = []
+    min-epochs-between-updates = 50
+  }
+  dev: {
+    # null means all metagraph ids are allowed
```

Comment:
I think we need to explicitly enable this setting to verify the functionality works.